docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-secretstore/snippets/dapr-secretstore-recipe.bicep

```
@@ -0,0 +1,29 @@
+import radius as radius
+
+@description('The ID of your Radius environment. Automatically injected by the rad CLI.')
+param environment string
+
+resource app 'Applications.Core/applications@2022-03-15-privatepreview' = {
+  name: 'dapr-secretstore-generic'
+  properties: {
+    environment: environment
+  }
+}
+
+//SAMPLE
+resource secretstore 'Applications.Link/daprSecretStores@2022-03-15-privatepreview' = {
+  name: 'secretstore-generic'
+  properties: {
+    environment: environment
+    application: app.id
+    recipe: {
+      // Name a specific Recipe to use
+      name: 'keyvault'
```

recommend renaming to something more generic sounding for now since we don't support KeyVault yet.

```suggestion
      name: 'secret-provider'
```